src/main/src/test/java/org/geoserver/wfsng/WfsCompatibilityTest.java

```diff
@@ -67,12 +67,14 @@ public void testWfsCompatitibility() throws IOException {
         }
                 
         assertTrue(store instanceof WFSDataStore);
-                
-        FeatureType type = ftInfo.getFeatureType();
-        
-        assertEquals("sf_archsites", type.getName().getLocalPart());        
         
-        assertEquals("sf_archsites", ftInfo.getFeatureSource(null, null).getName().getLocalPart());
-
+        try{
```

Comment:
Still wrong, there is a space between try and the curly brace, same at the end of the catch exception. As a general rule, always have a space before and/or after curly braces